bagua/torch_api/algorithms/decentralized.py

```diff
@@ -99,27 +101,29 @@ def hook(parameter_name, parameter):
 
     def init_post_backward_hook(self, bagua_module: BaguaModule):
         def hook():
-            for bucket in bagua_module.bagua_buckets:
-                for tensor in bucket.tensors:
-                    tensor.bagua_mark_communication_ready()
-
-            bagua_module._bagua_backend.wait_pending_comm_ops()
+            pass
 
         return hook
 
     def init_post_optimizer_step_hook(self, bagua_module: BaguaModule):
         def hook(optimizer: torch.optim.Optimizer):
```

Comment:
need to consider the case where there are multiple optimizers

Reply:
current implementation will mark all buckets ready when the first optimizer'`.step` gets called

Reply:
fixed